notebooks/README.md

```diff
@@ -0,0 +1,47 @@
+# Recommender notebooks
+
+This folder contains examples and best practices, written in Jupyter notebooks, for building recommendation systems.
+
+## Summary
+
+The following summarizes each directory of the best practice notebooks.
+
+| Directory | Is Azure required | Description |
+| --- | --- | --- |
+| [00_quick_start](./00_quick_start)| No | Quick start notebooks that demonstrate workflow of developing a recommender by using an algorithm in local environment|
+| [01_prepare_data](./01_prepare_data) | No | Data preparation notebooks for each recommender algorithm|
+| [02_model](./02_model) | No | Deep dive notebooks about model building by using various classical and deep learning recommender algorithms|
+| [03_evaluate](./03_evaluate) | No | Notebooks that introduce different evaluation methods for recommenders|
+| [04_model_select_and_optimize](04_model_select_and_optimize) | Yes | Best practice notebooks for model tuning and selecting by using Azure Machine Learning Service and/or open source technologies|
+| [05_operationalize](05_operationalize) | Yes | Operationalization notebooks that illustrate an end-to-end pipeline by using a recommender algorithm for a certain real-world use case scenario|
+
+## On-premise notebooks
+
+The notebooks that do not require Azure can be run out-of-the-box on any Linux machine, where an environment is properly
+set up by following the [instructions](../SETUP.md). 
+
+**NOTE** some of the notebooks may rely on heterogeneous computing instances
+like a cluster of CPU machines with Spark framework installed or machines with GPU devices incorporated. It is therefore recommended
+to run these notebooks on an Azure Data Science Virtual Machine, where single-node Spark and/or GPU devices are pre-configured.
+
+## Azure-enhanced notebooks
+
+Azure products and services are used in certain notebooks to enhance the development-efficiency of recommender system in scale.
+To successfully run these notebooks, the users **need an Azure subscription**.
+The Azure products featured in the notebooks include
+
+* [Azure Machine Learning Service](https://azure.microsoft.com/en-us/services/machine-learning-service/) - it is used for model building (e.g., hyperparameter tuning), model serving, etc. Azure
+Machine Learning Service is used intensively across various notebooks - particularly, it is used for hyperparameter optimization
```

Comment:
We also use it in some notebooks for:
- Tracking and monitoring metrics to enhance the model creation process
- Scaling up and out on Compute like DSVM and Azure Machine Learning Compute
- Deploying a web service to AKS

Reply:
Azure Machine Learning service is a cloud service that you use to train, deploy, automate, and manage machine learning models, all at the broad scale that the cloud provides.

Reply:
This is super great! 
+1 on wutaomsft comment: Instead of the requires Azure column, what about a Runs on your local machine column?